products/silverstripe.md

```diff
@@ -74,6 +74,41 @@ releases:
     latest: "4.10.11"
     latestReleaseDate: 2022-11-21
 
+-   releaseCycle: "3.7"
+    releaseDate: 2018-06-08
+    support: true
```

Comment:
```suggestion
    support: false
```

The release being eol.

Reply:
Sorry, I understood the contributing guidelines differently. 

```
# End of active support date (optional if activeSupportColumn is false, else mandatory).
# This is where bug fixes usually stop coming in.
# Use valid dates, and do not add quotes around dates.
# Alternatively, set to true|false if the date has not been decided yet.
support: 2018-01-31
```

Being the 'end of support date' and `false` being 'not decided yet', I thought `true` means 'end of support, but no date available'?

Reply:
This is counter intuitive and should be called eos (meaning end of support), to be aligned with the eol field IMO.

If support is a date, it means this is the end of support date. Else it can be a boolean, and in this case if it's true, it means the release is still supported. Otherwise (if it's false), it means the release is not supported anymore.

Reply:
ok, so maybe the contributing guidelines should be amended to reflect and explain this boolean state?